README.md

```diff
@@ -365,6 +365,49 @@ private Binding<HttpStatus> problems(final Condition<HttpStatus> condition) {
 }
 ```
 
+#### Template Routing
+
+Routing and response handling is sometimes pretty similar for many request, and can even be handle using the same
+defined routing logic for different requests. To enable reuse of this routing logic, Riptide supports the creation
+of nested *template routing*, that can be reused for dispatching different request. The following example defines
+a nested hierarchy of routing templates:
+
+```java
+static final Router<MediaType> CONTENT_ROUTER = Router.create(contentType(),
```

Comment:
> No, it is not. It is stated on the idea, that definitions can be much smaller, if you once define the routing template and than change/reuse it for many different requests points.

Ok, but do we have the evidence that this is a common case? Do you intend to share routing trees across gateways? Or inside a gateway across different kind of HTTP requests to the same remote service?

I feel like externalizing the whole tree removes a lot of visibility. I have the same problem with constants in general, one always has to look them up while reading/understanding the code. Isn't embedding re-used parts (#73) enough for this?